tests/integration/test_fim/test_registry/test_ignore/test_ignore_registry.py

```diff
@@ -8,66 +8,45 @@
 
 import pytest
 
-from wazuh_testing.fim import LOG_FILE_PATH, callback_ignore, create_registry, delete_registry, registry_parser
+from wazuh_testing.fim import LOG_FILE_PATH, callback_ignore, create_registry, delete_registry, registry_parser, \
+        KEY_WOW64_32KEY, KEY_WOW64_64KEY, generate_params, callback_detect_event, check_time_travel, modify_registry_value
```

Comment:
`delete_registry` is imported but unused. Also, line 12 exceeds 120 characters.
```suggestion
from wazuh_testing.fim import LOG_FILE_PATH, callback_ignore, create_registry, registry_parser, KEY_WOW64_32KEY, \
    KEY_WOW64_64KEY, generate_params, callback_detect_event, check_time_travel, modify_registry_value
```